indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -269,12 +277,11 @@ class BookRoomModal extends React.Component {
         if (!room) {
             return null;
         }
-
+        const occurrencesNumber = availability && availability.dateRange.length;
```

Comment:
I'm not sure if that's the best way to count occurrences in this component. What do you think?

Reply:
`numOccurrences` or `occurrenceCount`, the current name sounds like some number assigned to a set of occurrences.

Regarding the way to count them.. seems OK to me. At least I can't think of a better way right now.